android/libraries/rib-android/src/main/kotlin/com/uber/rib/core/RibActivity.kt

```diff
@@ -96,10 +96,9 @@ abstract class RibActivity :
     initViewTreeOwners()
     val rootViewGroup = findViewById<ViewGroup>(android.R.id.content)
     _lifecycleFlow.tryEmit(createOnCreateEvent(savedInstanceState))
-    val wrappedBundle: Bundle? =
-      if (savedInstanceState != null) Bundle(savedInstanceState) else null
     router = createRouter(rootViewGroup)
     router?.let {
+      val wrappedBundle: Bundle? = savedInstanceState?.let(::Bundle)
```

Comment:
`wrappedBundle` is used only if router is not null. So, moved to this let.

Reply:
We could probably go further and
```
    createRouter(rootViewGroup).let { r ->
      router = r
      val wrappedBundle = savedInstanceState?.let(::Bundle)
      r.dispatchAttach(wrappedBundle)
      rootViewGroup.addView(r.view)
      RibEvents.emitRouterEvent(RibEventType.ATTACHED, r, null)
    }
```

To avoid the unnecessary null check (router will never be null at this point)